app.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="h-screen flex flex-col font-sans text-primary-text bg-primary-bg">
+    <div class="h-screen flex flex-col font-sans text-primary-text bg-primary-bg theme-neon">
```

Comment:
```suggestion
    <div class="h-screen font-sans text-primary-text bg-primary-bg theme-neon">
```

Reply:
The `flex flex-col` doesn't do anything and should be removed.

Reply:
@theyokohamalife  It does :) I have the `<NuxtPage>` set to `flex-1` so it will expand to fill the screen. 
Without the flex flex-col, it looks like this :( 

<img width="1387" alt="image" src="https://user-images.githubusercontent.com/4411499/211196124-b96b447f-a4f0-4a02-b50b-155b124137a6.png">